src/components/Tokens/TokensBanner.tsx

```diff
@@ -71,26 +75,28 @@ export default function TokensBanner() {
   }
 
   return (
-    <PopupContainer show={showTokensPromoBanner} onClick={navigateToExplorePage}>
-      <Header>
-        <HeaderText to={'/tokens'}>
-          <Trans>Explore Top Tokens on Uniswap</Trans>
-        </HeaderText>
-        <X
-          size={20}
-          color={theme.textSecondary}
-          onClick={(e) => {
-            e.preventDefault()
-            e.stopPropagation()
-            setShowTokensPromoBanner(false)
-          }}
-          style={{ cursor: 'pointer' }}
-        />
-      </Header>
+    <BackgroundColor>
```

Comment:
won't this persist if the banner isn't shown?

Reply:
![image](https://user-images.githubusercontent.com/5773490/195491130-8630757b-3c21-4753-aa41-58593d58a701.png)

gah! good catch! I assumed it was being conditionally rendered by the parent